cmd/tf-operator.v1/main.go

```diff
@@ -31,6 +35,15 @@ func init() {
 	log.AddHook(filenameHook)
 }
 
+func startMonitoring() {
+	go func() {
+		monitoringPort := os.Getenv("MONITORING_CLIENT_PORT") //TODO (krishnadurai): remove with static port
```

Comment:
To be removed with a hardcoded port number or an optional server flag.
Please suggest.

Reply:
For now, you can keep a server flag with defaults.  https://github.com/kubeflow/tf-operator/blob/master/cmd/tf-operator.v1/app/options/options.go#L24